src/features/SingleHacker/SingleHackerModal.tsx

```diff
@@ -45,7 +50,11 @@ class SingleHackerModal extends React.Component<IModalProps, IModalState> {
     return (
       // tslint:disable-next-line
       <div onKeyDown={(e) => this.handleKeyPress(e)}>
-        <Button type="button" onClick={this.handleOpenModal}>
+        <Button
+          type="button"
+          onClick={this.handleOpenModal}
+          variant={ButtonVariant.Primary}
+        >
           See more
```

Comment:
super nit: can we center this button in the cell? Looks a bit off being aligned left